src/util.rs

```diff
@@ -7,6 +7,115 @@
 // Media Patent License 1.0 was not distributed with this source code in the
 // PATENTS file, you can obtain it at www.aomedia.org/license/patent.
 
+// Imported from clap, to avoid to depend directly to the crate
```

Comment:
Is there really no better way to do this?

Reply:
that's the simplest way since `Display`, `FromStr` and a mean to iterate values would be needed. Incidentally it provides also the same enum iterator capability we use for other enums so we could use it there as well and remove another crate :)

Reply:
I see. I guess we could have a separate "external" settings struct or something too. I guess this is okay.